doc/doxygen/src/css/riot.less

```diff
@@ -266,6 +266,18 @@ table.directory {
 .navpath li.navelem a {
   font-family: @font-family-base;
 }
+.memtitle {
+  background: none;
```

Comment:
@miri64: I fiddled around with that a bit yesterday evening  and found that to get rid of this, just write `display: none;`. Which looks better (IMHO).